### PR TITLE
Feature/hack config import

### PR DESCRIPTION
**Pull request by dblane-digicatapult**
Templates the config.js in helm
Replaces the config vars used in the application

### CODE REVIEW

File: helm/inteli-demo/files/config.js.tpl [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    Window.config = {}
+  }
+  
+  if (!Window.config.inteli) {
+    Window.config.inteli = {}
+  }
+  
+  for (const [key, value] of Object.entries(config)) {
+    Window.config.inteli[key] = value
+  }
```
Comment (mattdean-digicatapult) [inline helm/inteli-demo/files/config.js.tpl:13]: fix indentation

Comment (mattdean-digicatapult) [inline helm/inteli-demo/files/config.js.tpl:13]: why the loop here?

File: helm/inteli-demo/templates/configmap.yaml [hunk 2]
```diff
@@ -5,4 +5,4 @@ metadata:
   labels:
     {{- include "inteli-demo.labels" . | nindent 4 }}
 data:
-  applicationConfig.json: {{ .Values.config.applicationConfig }}
+  config.js: {{- tpl (.Files.Get "files/config.js.tpl") . }}
```
Comment (mattdean-digicatapult) [inline helm/inteli-demo/templates/configmap.yaml:8]: I like this. With the hash on the deplyment I assume this means whenever the config is updated so will the deployment

File: helm/inteli-demo/values.yaml [hunk 3]
```diff
@@ -3,7 +3,9 @@ config:
   port: 80
   logLevel: info
   enableLivenessProbe: true
-  applicationConfig: {}
+  applicationConfig:
+    foo: bar
+    blah: reeeee
```
Comment (mattdean-digicatapult) [inline helm/inteli-demo/values.yaml:8]: not sure these are the exact keys we need? Isn't it:
```
  // authDomain
  // clientID
  // custAuthClientID
  // t1AuthClientID
  // authAudience
  // apiHost
  // apiPort
  // substrateHost
  // substratePort
  // vitalamDemoPersona
```
based on the default `config.js`

File: public/config.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  // authDomain
+  // clientID
+  // custAuthClientID
+  // t1AuthClientID
+  // authAudience
+  // apiHost
+  // apiPort
+  // substrateHost
+  // substratePort
+  // vitalamDemoPersona
```
Comment (mattdean-digicatapult) [inline public/config.js:11]: Can we have the defaults in here instead of in the `env.js`?

**Review by mattdean-digicatapult — CHANGES_REQUESTED** [review]

File: helm/inteli-demo/templates/configmap.yaml (hunk 2 above)
Comment (dblane-digicatapult) [reply helm/inteli-demo/templates/configmap.yaml:8]: exactly.

File: helm/inteli-demo/values.yaml (hunk 3 above)
Comment (dblane-digicatapult) [reply helm/inteli-demo/values.yaml:8]: these were just draft values.  I'll put some sensible defaults in place.

File: helm/inteli-demo/files/config.js.tpl (hunk 1 above)
Comment (dblane-digicatapult) [reply helm/inteli-demo/files/config.js.tpl]: @danielcooperxyz ?

**Pull request merged**